app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+class CarWashStation:
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: float,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
 
+    def serve_cars(self, cars: list):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += (car.comfort_class *
+                           (self.clean_power - car.clean_mark) *
+                           (self.average_rating/self.distance_from_city_center))
+                car.clean_mark = self.clean_power
+        return round(income, 1)
 
-class CarWashStation:
-    # write your code here
-    pass
+    def calculate_washing_price(self, car: str):
+        self.car = car
+        wash_cost = (car.comfort_class *
+                  (self.clean_power - car.clean_mark) *
+                  (self.average_rating / self.distance_from_city_center))
+        return wash_cost
+
+    def wash_single_car(self, car: str):
```

Comment:
The parameter `car` should be of type `Car`, not `str`. This method is supposed to wash a `Car` object.